service/src/main/java/greencity/mapping/AddLocationDtoMapper.java

```diff
@@ -0,0 +1,25 @@
+package greencity.mapping;
+
+import greencity.dto.AddLocationDto;
+import greencity.dto.AddLocationTranslationDto;
+import greencity.entity.user.Location;
+import greencity.entity.user.LocationTranslation;
+import org.modelmapper.AbstractConverter;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class AddLocationDtoMapper extends AbstractConverter<Location, AddLocationDto> {
+    @Override
+    protected AddLocationDto convert(Location source) {
+        List<LocationTranslation> locationTranslations = new ArrayList<>(source.getLocationTranslations());
```

Comment:
I don't understand why do you need another collection `locationTranslations`. Can't you just loop 
`source.getLocationTranslations()` directly?